src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -345,30 +345,30 @@
   "item.minecolonies.golden_bread": "Golden Bread",
   "item.minecolonies.chorus_bread": "Chorus Fruit Bread",
 
-  "item.minecolonies.scroll_tp": "Ultrasafe colony teleport scroll",
+  "item.minecolonies.scroll_tp": "Ultrasafe Colony Teleport Scroll",
   "item.minecolonies.scroll_tp.tip": "Safely teleports you to the set colony",
   "item.minecolonies.scroll.colony.tip": "Colony: %s",
   "item.minecolonies.scroll.colony.none": "None, set one with sneak + right-click",
-  "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
+  "item.minecolonies.scroll_area_tp": "Ultrasafe Colony Group Teleport Scroll",
+  "item.minecolonies.scroll_area_tp.tip": "Safely teleports you and nearby players to the colony",
   "item.minecolonies.scroll_buff": "Sacred Scroll of Regeneration",
   "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",
-  "item.minecolonies.scroll_guard_help": "Spatial guard reinforcement scroll",
-  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time, wololo!",
-  "item.minecolonies.scroll_highlight": "Worker-where-are-you scroll",
-  "item.minecolonies.scroll_highlight.tip": "Sneak-Use on a minecolonies building, to highlight and speed up its workers",
+  "item.minecolonies.scroll_guard_help": "Spatial Guard Reinforcements Scroll",
+  "item.minecolonies.scroll_guard_help.tip": "Calls your Guards to help you out for a limited time",
+  "item.minecolonies.scroll_highlight": "Worker-Where-Are-You Scroll",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + right-click on a MineColonies building to highlight and speed up its worker(s)",
   "minecolonies.scroll.nopermission": "You do not have the required permissions for this colony.",
-  "minecolonies.scroll.needcolony": "Please register this to a colony first with shift + right-clicking a building.",
-  "minecolonies.scroll.noguardbuilding": "Please register this to a guard building by shift + right-clicking it.",
-  "minecolonies.scroll.wrongdim": "Can only be used in the same dimension as the colony.",
-  "minecolonies.scroll.registered": "Registered scroll with colony %s",
+  "minecolonies.scroll.needcolony": "Please register this scroll to a colony first by sneaking + right-clicking it on a MineColonies building.",
```

Comment:
Should probably avoid calling it "Minecolonies" building for immersion but just a "colony building"